test-vectors/json/sapling_zip32_hard.json

```diff
@@ -0,0 +1,8 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/sapling_zip32.py"],
```

Comment:
This URI is now incorrect.

Reply:
That's been incorrect for a while, both due to internal refactoring to `poetry` and the recent repository move. This should be fixed separately across the entire test vector corpus.